monailabel/transform/post.py

```diff
@@ -124,6 +134,15 @@ def __call__(self, data):
                 resizer = Resize(spatial_size=spatial_size, mode=self.mode[idx])
                 result = resizer(result, mode=self.mode[idx], align_corners=self.align_corners[idx])
 
+            if self.invert_orient:
+                # Undo Orientation
+                orig_affine = meta_dict["original_affine"]
```

Comment:
what if "original_affine" not exists? in that case you want to skip undo-orientation

Reply:
Good point. Just updated this. Hope this makes sense :)